### ISSUE TITLE
Create 2021-2022 Academic Year

### ISSUE THREAD

**Issue by arana2** [smiths/MEASURE-Doc#273]
- [x] Confirm Previous Year Data Archived

- [x] Execute ETL Job to Create New Year
   
  - [x] Open New Academic Year, insert populated AcademicYearTemplate.csv

  - [x]  Run ETL Job: New Academic Year
  
  - [x]  Confirm 2021-2022 academic year has been created

**Comment by arana2** [comment]
- Received an ETL Service error when running the "Create a New Year" process
- Notified Vena Support and they are currently investigating
  -  https://support.venasolutions.com/hc/en-us/requests/56795
- Will resume activities once this issue has been resolved

**Comment by arana2** [comment]
- Issue has been resolved and able to successfully create a new academic year
- Root Cause: One of the LID's exceeded the maximum character limit.  Note, I don't recall having an LID character limit and if we did, we would have documentation (or know this in advance).  Also, Vena does not know the maximum character limit and is still investigating.
- Work around: Reduce the number of characters for the LID in question (ENGPHYS 4A04)
  - We reached out to the Undergrad Administrative Assistant to let them know the issue and requested for a summarized version (less than 1024 characters)
  - Export the LID's in a CSV format
  - Update the LID in question and upload the CSV back to Vena
  - Delete the corrupted academic year and re-run the 'New Academic Year' ETL Process
